ports/psoc6/boards/CY8CPROTO-062-4343W/makefile_mtb.mk

```diff
@@ -70,7 +70,7 @@ mpy_define_mtb_vars: mpy_mtb_build
 #	$(info CXXFLAGS : $(CXXFLAGS))
 	$(eval LDFLAGS += -mcpu=cortex-m4 --specs=nano.specs -mfloat-abi=softfp -mfpu=fpv4-sp-d16 -mthumb -ffunction-sections -fdata-sections -ffat-lto-objects -g -Wall -pipe -Wl,--gc-sections -T$(MPY_DIR_OF_MTB_ADAPTER_MAKEFILE)/bsps/TARGET_APP_CY8CPROTO-062-4343W/COMPONENT_CM4/TOOLCHAIN_GCC_ARM/linker.ld -Wl,-Map,$(BUILD)/firmware.map -Wl,--start-group -Wl,--end-group -Wl,--print-memory-usage)
 #	$(info LDFLAGS : $(LDFLAGS))
-	$(eval QSTR_GEN_CFLAGS += $(INC))
+	$(eval QSTR_GEN_CFLAGS += $(INC) $(CFLAGS))
```

Comment:
@jaenrig-ifx : Do we need all the `CFLAGS` for compilation of the `QSTR` headers? I thought the `CFLAGS` are mainly for the compilation of the main source code(MPY core and PSoC6 BSPs etc)